### DISCUSSION TITLE
progress display options from stable

### DISCUSSION THREAD

**Discussion by moonfight1** [ppy/osu#31317] [Ideas] [answered]
![image](https://github.com/user-attachments/assets/b55926c3-83ee-41b3-9056-4c3a3658220f)

**Comment by Stoppedpuma** [comment]
These are available in the skin editor while under gameplay.

[editor.webm](https://github.com/user-attachments/assets/e18cc99b-da09-4073-82e5-b23345483381)

**Reply by moonfight1** [reply]
no i mean in custom skins not argon, i know those exist

**Reply by Stoppedpuma** [reply]
The same options exist for custom skins. If you mean actually skinning the element then you can just drag an image on top of it and it should look the same as stable.

**Reply by moonfight1** [reply]
![image](https://github.com/user-attachments/assets/88d9a8e3-3a63-4e08-ae31-b7b63301e9d4)
this is on stable, is it possible to make the progress display look like this on lazer?

**Reply by Stoppedpuma** [reply, marked as answer]
Yes, here's the argon progress bar with a few customisations to look similar:

![image](https://github.com/user-attachments/assets/1a1b48c7-0a07-4dfa-87e6-85c87826a93c)

![image](https://github.com/user-attachments/assets/86193f46-152e-44c0-918c-250e363c0a04)

**Reply by moonfight1** [reply]
oh thanks, i didnt know they had settings, guess that shows that the lazer options are more than stable's
